Gruntfile.js

```diff
@@ -88,6 +90,47 @@ module.exports = function( grunt ) {
 				src: HH_JS
 			}
 		},
+		postcss: {
+			options: {
+				map: false,
+				processors: [
+					autoprefixer({
+						browsers: [
+							'> 1%',
+							'ie >= 11',
+							'last 1 Android versions',
+							'last 1 ChromeAndroid versions',
+							'last 2 Chrome versions',
+							'last 2 Firefox versions',
+							'last 2 Safari versions',
+							'last 2 iOS versions',
+							'last 2 Edge versions',
+							'last 2 Opera versions'
+						],
+						cascade: false
+					})
+				],
+				failOnError: false
+			},
+			helphub: {
+				expand: true,
+				src: 'themes/helphub/sass/style.css'
```

Comment:
This might be typo. There is no `.css` file in `sass` folder. Did you mean `themes/helphub/sass/style.scss`?

Reply:
Great catch, fixed in https://github.com/Kenshino/HelpHub/pull/133/commits/ffdc7d172cf9cb0994eca53af6858d2569bf43be